bibtex-actions.el

```diff
@@ -161,12 +161,10 @@ means no action."
   :type '(choice (const :tag "Prompt" 'prompt)
                  (const :tag "Ignore" nil)))
 
-(defcustom bibtex-actions-embark-dwim t
-  "Whether to run the default action on citation keys at point.
-If non-nil, call the default embark action through `embark-dwim'.
-If nil, prompt the user for an action through `embark-act'."
+(defcustom bibtex-actions-at-point-function 'embark-dwim
```

Comment:
```suggestion
(defcustom bibtex-actions-at-point-function nil
```

Reply:
I'm unsure the best approach.

We could do this change, and reset on `bibtex-actions-org-cite`?

Reply:
> We could do this change, and reset on `bibtex-actions-org-cite`?

I made this change. I think that's my last commit for the day.